src/rpc/client/ts/tests/specs/common.ts

```diff
@@ -59,6 +59,9 @@ export async function check_callback_count(
   expected_count: number,
   millisecond_delay?: number
 ) {
+  const do_check: boolean =
```

Comment:
@scholarsmate Can you update to this check to be

```ts
const do_check: boolean = process.env.OMEGA_EDIT_CLIENT_CALLBACK_COUNT_CHECKS
    ? process.env.OMEGA_EDIT_CLIENT_CALLBACK_COUNT_CHECKS === '0'
    : false
```
so that by default if that variable isn't set it won't run and if it is set to '0' it will run the check?